UIKitRMDZ/UIKitRMDZ/Cells/NewPostCell.swift

```diff
@@ -0,0 +1,334 @@
+// NewPostCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка последнего опубликованного поста
+final class NewPostCell: UITableViewCell {
+    // MARK: - Constants
+
+    enum Constants {
+        static let coment = "Комментировать..."
+        static let tenMinsAgo = "10 минут назад"
+    }
+
+    static let identifier = "NewPostCell"
+
+    // MARK: - Private properties
+
+    private let imageViewPost = UIImageView()
+    private let secImageViewPost = UIImageView()
+    private let thirdImageViewPost = UIImageView()
```

Comment:
Название элемента желательно писать в конце переменной в таком виде
```suggestion
    private let thirdPostImageView = UIImageView()
```